src/components/MessageInput/TextInput.vue

```diff
@@ -119,11 +122,33 @@ export default {
                 return `<span class="channel ${item.channel.type}"><span class="channel-name ">${item.value}</span></span>`
               }
             },
-            source: function (searchTerm, renderList, mentionChar) {
+            source: async function (searchTerm, renderList, mentionChar) {
```

Comment:
(keep in mind) -- This will need to be refactored when TipTap migration gets merged.

Reply:
But still, add a test or two, to make sure it works